tpgtools/README.md

```diff
@@ -79,3 +79,100 @@ For example, override entries will look like the following:
   details:
     description: Custom description here.
 ```
+
+#### Samples
+
+We will autoingest samples from the dcl, however we currently must
+manually fill the substitutions for these samples.
+
+You may need to re-serialize `serialization.go` if you are adding newer resources.
+To do this
+```
+cd tpgtools
+go get -u github.com/GoogleCloudPlatform/declarative-resource-client-library
+make serialize
+```
+
+To do so, first create a folder in the relevant product
+```
+$ cd overrides/{{product}}/
```

Comment:
This is now in a different place, `samples/{{product}}` right?

Reply:
I am moving `samples` to `overrides` with this pr